### DISCUSSION TITLE
Does Deepgram API Key expires?

### DISCUSSION THREAD

**Discussion by jswinnen** [deepgram/community#641] [General help] [answered]
I'm working on a project and I'd like to know if Deepgram API key has an expiration date or not.

Thanks!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@jswinnen 👋  It depends. You can set a key to expire or leave it with no expiration date. You can read more about our API key expiration [here](https://developers.deepgram.com/docs/create-additional-api-keys).
